src/main/java/de/slimecloud/slimeball/features/alerts/youtube/Youtube.java

```diff
@@ -40,43 +40,44 @@ public Youtube(String[] keys, SlimeBot bot, YoutubeConfig config) {
 	}
 
 	public void startListener() {
-		int delay = config.getUpdateRate();
-
-		bot.getExecutor().scheduleAtFixedRate(() -> {
-			try {
-				check();
-			} catch (Exception e) {
-				logger.error("Failed to check for new video", e);
-			}
-		}, 0, delay, TimeUnit.SECONDS);
+		Map<String, Integer> channels = config.getYoutubeChannels();
+
+		channels.forEach((channel, delay) -> {
+			bot.getExecutor().scheduleAtFixedRate(() -> {
+				try {
+					check(channel);
+				} catch (Exception e) {
+					logger.error("Failed to check for new video for channel " + channel, e);
+				}
+			}, 0, delay, TimeUnit.SECONDS);
+		});
 	}
 
-	private void check() throws IOException {
-		Set<Video> videos = getLastVideo(5);
+	private void check(String youtubeChannelId) throws IOException {
+		Set<Video> videos = getLastVideo(youtubeChannelId, 5);
 
-		Collection<String> known = bot.getIdMemory().getMemory("youtube");
+		Collection<String> known = bot.getIdMemory().getMemory("youtube-" + youtubeChannelId);
 		List<String> newIds = new ArrayList<>();
 
 		for (Video video : videos) {
 			if (!known.contains(video.id())) {
-				new YoutubeVideoEvent(video).callEvent();
+				new YoutubeVideoEvent(youtubeChannelId, video).callEvent();
 				newIds.add(video.id());
 			}
 		}
 
-		bot.getIdMemory().rememberIds("youtube", newIds);
+		bot.getIdMemory().rememberIds("youtube-" + youtubeChannelId, newIds);
```

Comment:
This is not required. Video ids are unique over all of youtube so the channel id is not required as extra specifier